### PR TITLE
Add analysis type to metadata and change analysis to analysis_type

### PR DESCRIPTION
**Pull request by UsamaEquinorAFK**

### CODE REVIEW

File: src/isar/storage/utilities.py [hunk 1]
```diff
@@ -26,6 +26,7 @@ def construct_paths(inspection: Inspection, mission: Mission) -> Tuple[Path, Pat
 def construct_metadata_file(
     inspection: Inspection, mission: Mission, filename: str
 ) -> bytes:
+    print(inspection.metadata.additional)
```
Comment (Eddasol) [inline src/isar/storage/utilities.py:29]: Remove print

Comment (UsamaEquinorAFK) [reply src/isar/storage/utilities.py]: Done - thought I removed it

**Comment by UsamaEquinorAFK** [comment]
Should we update remainig fields from flotilla in metadatafile as well ?  @oysand

**Comment by UsamaEquinorAFK** [comment]
Should we update the remaining fields in the metadafile from flotilla as well oystein ? @oysand

**Review by Eddasol — APPROVED** [review]
LGTM

**Comment by UsamaEquinorAFK** [comment]
> LGTM

The commits added after your approval are the same but rebased on the lattest commits from isar upstream main.

**Pull request merged**